tests/e2e/lib.sh

```diff
@@ -330,6 +333,9 @@ deploy_sensor_via_operator() {
        kubectl -n stackrox set env deployment/sensor ROX_PROCESSES_LISTENING_ON_PORT="${ROX_PROCESSES_LISTENING_ON_PORT}"
        kubectl -n stackrox set env ds/collector ROX_PROCESSES_LISTENING_ON_PORT="${ROX_PROCESSES_LISTENING_ON_PORT}"
     fi
+    if [[ "${ROX_COMPLIANCE_ENHANCEMENTS}" == "true" ]]; then
```

Comment:
It should no longer be necessary to set the flag on the sensor side after https://github.com/stackrox/stackrox/pull/8889